rpcserverhandlers_test.go

```diff
@@ -1933,6 +1950,8 @@ func TestHandleEstimateFee(t *testing.T) {
 }
 
 func TestHandleEstimateSmartFee(t *testing.T) {
+	t.Parallel()
+
```

Comment:
Srry, I snuck this one past you last review.

Reply:
Noticed that as well afterward, thanks.